index.html

```diff
@@ -76,25 +100,30 @@ <h2 id="terminology"><a class="toc-anchor" href="#terminology">§</a> Terminolog
 <td style="text-align:left">Unique ID string and PKI metadata document format for describing the cryptographic keys and other fundamental PKI values linked to a unique, user-controlled, self-sovereign identifier in a target system (i.e. blockchain, distributed ledger).</td>
 </tr>
 <tr>
-<td style="text-align:left">Subject</td>
+<td style="text-align:left">Holed</td>
```

Comment:
typo: Holder

Reply:
This wasn't in my change set, and to change this, you'll need to do it in the markdown file. Can you submit a separate PR for this that changes the MD file?

Reply:
I went ahead and made this change in the PR feedback update